README.md

```diff
@@ -1,7 +1,7 @@
 # hera_cal
+[![](https://github.com/HERA-Team/hera_cal/workflows/Run%20Tests/badge.svg?branch=master)](https://github.com/HERA-Team/hera_qm/actions)
```

Comment:
Hey now, a little too much copy-paste here.

Reply:
bummer. let me decopy that.